web/fp_delete.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+$mid = $_GET["mid"];
+/*if ($_GET["ismon"] = "true")
+{ $ismon = true}else{$ismon = false};*/
+//$ismon = $_GET["ismon"];
+//$json = $_GET["json"];
+
+
+$file_handle = fopen('/etc/zm.conf', 'r');
```

Comment:
Why are we doing all this, instead of using the built in ZM functions that automatically do this? So that it can stand alone outside of a ZM install? Seems weird.  If /etc/zm/zm.conf exists, then you should be able to include the database and config.php includes.

Reply:
I started it as a stand alone function, so it can be used as a simplified user interface with some branding (colors, fonts, etc.) based in the floorplan and just for viewing purposes. Because of that, I tried to use the just the config file to get database parameters and then make it as  autonomous as possible, we can create a separate config file, but is up to what we want to do with it...